ietf/templates/nomcom/private_index.html

```diff
@@ -11,21 +11,21 @@ <h2 class="mt-3">Nomination status</h2>
     <table class="table table-sm table-striped table-hover tablesorter">
         <thead>
             <tr>
-                <th scope="col" data-sort="position">Position</th>
+                <th scope="col" data-sort="position">Po&shy;si&shy;tion</th>
```

Comment:
Wow, this makes the template almost unreadable to a human.
Perhaps there's a bit of template middleware somewhere that keeps a dictionary of good splits that a block of such things could be fed to? Or a templatetag at least.